pyproject.toml

```diff
@@ -458,7 +458,6 @@ module = [
     "sentry_plugins.base",
     "social_auth.migrations.*",
     "sudo.*",
-    "tests.sentry.api.endpoints.issues.test_organization_derive_code_mappings",
```

Comment:
I'm moving the test under `tests.sentry.issues.auto_source_config`, where all other related tests are located.